pkg/cmd/roachtest/registry/owners.go

```diff
@@ -29,15 +29,14 @@ const (
 	OwnerKV               Owner = `kv`
 	OwnerReplication      Owner = `replication`
 	OwnerAdmissionControl Owner = `admission-control`
-	OwnerObsInf           Owner = `obs-inf-prs`
```

Comment:
I see this as used in a few other files too. I see it in:

* pkg/cmd/roachtest/tests/acceptance.go
* pkg/cmd/roachtest/tests/kv.go